### ISSUE TITLE
Tekken 7 (389730) no longer works on 8.0-2 and experimental

### ISSUE THREAD

**Issue by Doomguy1364** [ValveSoftware/Proton#6860]
Compatibility Report
- Name of the game with compatibility issues: Tekken 7
- Steam AppID of the game:389730

System Information
OS: Arch Linux
KERNEL: 6.3.8-arch1-1
CPU: AMD Ryzen 5 5600X 6-Core
GPU: NVIDIA GeForce RTX 3070
GPU DRIVER: NVIDIA 535.54.0
Proton: Both Experimental and 8.0-2

I confirm:
That I haven't found an existing compatibility report for this game, similar to this one.

Symptoms:
Tekken 7 will no longer launch at all on newer protons. I have a sneaking suspicion this new client is to blame, there's no crashing or anything, It just lingers with a dead process tree until it is term/killed. This issue only started occurring since the forced release of new steam client. Proton 8.0-2 and Experimental worked flawlessly for Tekken 7 prior to the June 14th update to steam.

Reproduction:
Always, Attached log below on my most recent attempt.

[steam-389730.log](https://github.com/ValveSoftware/Proton/files/11780546/steam-389730.log)

**Comment by kisak-valve** [comment]
Hello @Doomguy136, please copy the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

These look like some possible lines of interest from your log:
```
dlmopen steamservice.so failed: steamservice.so: cannot open shared object file: No such file or directory
dlmopen libSDL3.so.0 failed: libSDL3.so.0: cannot open shared object file: No such file or directory
```

**Comment by Doomguy1364** [comment]
https://gist.github.com/Doomguy136/990c5fd16040f5a0800e4c8c962e9dba

**Comment by Doomguy136** [comment]
Side note and sorry for being off topic, but CSGO now crashes with the new steam overlay as well, printing the below error to the journal:
systemd-coredump[4777]: [🡕] Process 4633 (csgo_linux64) of user 1000 dumped core.

Stack trace of thread 4633:
#0 0x00007f363e9eb772 n/a (/home/doomguy/.local/share/Steam/ubuntu12_64/gameoverlayrenderer.so + 0x27772)
ELF object binary architecture: AMD x86-64


All other games I've tested with the new client work fine, only csgo and Tekken seem to be having issues so far, one being Linux native and the other through proton.

**Comment by Doomguy1364** [comment]
If you need anything else let me know!

**Comment by Doomguy1364** [comment]
This issue is fixed as of proton 8.0-3 and newest experimental (tested on both); You guys rock!
